### ISSUE TITLE
Bug: ERR_UNSUPPORTED_ESM_URL_SCHEME for undefined steps

### ISSUE THREAD

**Issue by yavulan** [vitalets/playwright-bdd#71]
**Given**

`playwright.config.ts`
```ts
import { defineConfig } from '@playwright/test';
import { defineBddConfig } from 'playwright-bdd';

const testDir = defineBddConfig({
    paths: ['sample.feature'],
    require: ['*.ts'],
});

export default defineConfig({
    testDir,
    reporter: 'html',
});

```

`sample.feature`

```feature
Feature: Home Page

    Scenario: Check title
        Given I am on home page
        Then I see in title "Playwright"
```

`steps.ts`

```ts
import { expect } from '@playwright/test';
import { createBdd } from 'playwright-bdd';

const { Given, When, Then } = createBdd();

Then('I see in title {string}', async ({ page }, keyword) => {
    await expect(page).toHaveTitle(new RegExp(keyword));
});
```

**When**
```shell
npx bddgen
```
**Then**

`ERR_UNSUPPORTED_ESM_URL_SCHEME` related to `loadSnippetBuilder`

**Expected behavior**

When I do have an undefined step, like `Given I am on home page` in this example, there shouldn't be an error. Or, at least, the error should contain human-readable message.

**Environment**

platform: win32
node: v18.16.0
playwright-bdd: v5.4.0
@playwright/test: v1.39.0
@cucumber/cucumber: v10.0.1
Playwright config file: playwright.config.ts
```
```

**Comment by vitalets** [comment]
Reproduced this bug, investigating.

**Comment by vitalets** [comment]
Fixed in dev, will be included in the nearest release.
Thanks for bringing this up!

**Comment by vitalets** [comment]
Landed in 5.5.0
